### DISCUSSION TITLE
Глупый вопрос

### DISCUSSION THREAD

**Discussion by rprovodenko** [bol-van/zapret#924] [General]
Я заранее прошу прощения за глупый вопрос, но: зачем это нужно?

Не легче ли купить дешевый VPS заграницей, поставить туда VPN и использовать его с какой-нибудь обфускацией? Это будет легче и эффективней (ведь, с zapret все равно невозможно обойти блокировку на уровне маршрутизации). 

Что я упускаю?

**Comment by alk2git** [comment]
Да, если Вам нужен доступ до ip-адресов, которые жестко заблокированы у провайдера, или Вы хотите попасть на сайты, которые сами закрыли доступ кому-либо из РФ, то без VPN не обойтись. Но!
zapret отлично справляется с задачей, когда доступ блокируется ТСПУ от РКН, этим "средствам" посылаются сбивающие их с толку пакеты, и дорога свободна! Во-первых, доступ оказывается из России, не надо заботиться о РФ-сайтах, которые не хотят пускать никого из-за бугра. Во-вторых, не ровен час очередные испытания Чебурнета - и доступ до дешевых VPS за границей может просто прекратиться. Что тогда? zapret!

**Reply by rprovodenko** [reply]
Спасибо большое за ответ!

1. Я скорее говорю про индивидуальную настройку VPN на VPS (VPS провайдеров сильно больше чем VPN)
2. Как я понимаю ТСПУ это DPI система? Если так, то докрутить к этой DPI системе блокировку по IP тривиально. Я это говорю к тому, что если смотреть в будущее, как я понимаю (?) риск того, что все равно к большинству (заграничных) ресурсов потребуется VPN больше, чем риск того, что все заграничные VPS провайдеры заблокируют. - поправьте меня если я не прав.

-----
Как я понимаю, у Вас есть представление, что Чебурнет будет развернут полностью как DPI система - не на уровне сети или даже не на уровне физических связей? 

Вы не могли бы поделиться, почему вы это так видите? 

Сейчас как я понимаю, они используют DPI для более точечных блокировок сервисов, которые используют общие IP адреса с другими сервисами, или же для блокировки конкретных страниц и тп - т.е. это как раз таки не Чебурнет. Чебурнет я вижу как именно блокировку иностранных IP адресов на уровне маршрутизатора (и тогда, Вы правы, VPN не поможет - но и DPI обход тоже не поможет).

Спасибо!

**Reply by bol-van** [reply]
У zapret есть свои области применения. VPS тоже могут начать банить.
Или смотреть, что у вас трафик идет на 1 IP и заносить его в стоп-лист

**Reply by alk2git** [reply]
Я думаю, что главная задача Чебурнета - сделать так, чтобы ничего не работало из того, что опасно для режима. И работало то, что режиму требуется. Так как множеству компаний нужен доступ до иностранных сервисов, прежде всего облачных, до собственных VPN - ведь в конце концов, VPN придуман, чтобы люди из локального офиса могли связываться с штаб-квартирой и наоборот, то "белый список" вряд ли реален. А вот обрубить по TCP доступ до веб-сайтов в широчайшем диапазоне ip-адресов при помощи ТСПУ задача очень даже выполнимая. Пусть себе всякие фрики ходят по нестандартным портам и нестандартным протоколам, их все равно мало, а у 99% процентов населения отрубится иностранный сегмент http. Ну так свой же есть! 
По такой логике не надо городить огород с маршрутизацией, DNS, всякими прочими опасными хренями. Интернет будет работать, но при этом http(s) будет только отечественный. Не знаю, возможно, у цензоров есть и сценарий Б, когда придется отрубать всё. Но это приведет к хаосу в бизнесе, к срыву контрактов, обрыву пресловутых "логистических цепочек", если не мятежу, то хотя бы к итальянской забастовке.

**Comment by TurboKirill** [comment]
Не легче. Во-первых, потеря на скорости. Во-вторых, российский Ютуб без рекламы) А если зайдешь под иностранным IP, то замучаешься рекламу смотреть)) Ну для оплаты на иностранных сайтах, конечно, приходится всё равно VPN юзать.
